primitives/primproc/batchprimitiveprocessor.cpp

```diff
@@ -1384,8 +1392,18 @@ void BatchPrimitiveProcessor::executeTupleJoin()
             //else
             // cout << "j != joinerCount\n";
         }
+        // If we've accumulated more than 1048576 (2^20)_ of resultCounts, cut off processing. The caller will
+        // restart to continue where we left off.
+        if (resultCount >= 1048576)
```

Comment:
Nit: It would be nice to have a named constant for this constant 1048576. And in other places in this code.

Reply:
+1

Reply:
Done